docs/try/next-steps.md

```diff
@@ -0,0 +1,244 @@
+
+As soon as the `witnet_node` container is up, it will do the following things in order:
+
+1. Try to **open connections to other nodes** in the network. It needs 8 "outbound" connections. This should take from several seconds to a few minutes.
+2. Discover what is the **tip of the block chain**, and **download all the blocks** from that chain. This can take from several minutes to several hours. The synchronization time depends heavily on how long the block chain is, but also on your Internet bandwidth, CPU speed, memory size and speed, and storage drive write throughput.
+3. Go into **_Synced_ status**. In Synced status, your node will **validate transactions and blocks** in real time, and it will try itself to **propose block candidates** and participate in **resolving _data requests_**.
+
+## What to expect from your node's balance and reputation
+
+- Getting your first block proposal accepted by the network and minting your first wit tokens is not easy, and can take from a few minutes up to 48 hours (or more!) due to the probabilistic nature of the [cryptographic sortition] algorithm that rules the system.
+- As with minting blocks, being assigned a request for the first time can take some time: from several minutes up to 48 hours (or more!). Once you have resolved at least one request, your node will earn reputation and it will start getting assignments more often.
+
+!!! warning "Don't panic"
+    Note that it is **perfectly normal** for a node to show 0 "balance", "reputation", "blocks included" or "accepted commits" for the first days of it being up.
+    Please be patient, new identities in the system are subject to a slow start for critical security reasons.
+
+## Monitoring your node's progress
+
+Here are some useful commands that you can use to keep track of how your node is performing in the network.
+A complete documentation of all the CLI methods is available in the [node operator docs][CLI]. 
+
+### nodeStats 
+
+Among other information, this shows the synchronization state of your node, as well as counters for proposed and accepted blocks and participations in resolving data requests ("commitments").
+
+```console tab="Docker"
+docker exec  witnet_node witnet node nodeStats
+```
+
+```console tab="Cargo"
+cargo run --release -- node nodeStats
+```
+
+```console tab="Binary"
+witnet node nodeStats
+```
+
+### balance
+
+The `balance` command will print your node's current balance.
+
+```console tab="Docker"
+docker exec witnet_node witnet node balance
+```
+
+```console tab="Cargo"
+cargo run --release -- node balance
+```
+
+```console tab="Binary"
+witnet node balance
+```
+
+
+### reputation
+
+The `reputation` command will print your node's current reputation score.
+
+```console tab="Docker"
+docker exec witnet_node witnet node reputation
+```
+
+```console tab="Cargo"
+cargo run --release -- node reputation
+```
+
+```console tab="Binary"
+witnet node reputation
+```
+
+
+!!! tip "Reputation is tricky"
```

Comment:
Maybe we could add that identity might appear as *not active*, which is perfectly normal in case the node wasn't lucky to mine a block or participate in a data request.